### ISSUE TITLE
New cstore_fdw releases

### ISSUE THREAD

**Issue by gabitoju** [citusdata/cstore_fdw#171]
Hi everyone. There hasn't been a release of cstore_fdw for a while now.
Is there a road map of new features or a release planed for the short term?

Thanks

**Comment by mtuncer** [comment]
we haven't been very active lately. 
use now postgres internal storage is checked into develop branch. This change enables streaming replication on cstore_fdw tables.

There are some pending task to address transaction support and backward compatibility. We would be releasing cstore_fdw 2.0 after addressing them.

Is there a particular feature you would be interested in ?

**Comment by gabitoju** [comment]
Hey @mtuncer, thanks for the reply.

I think this https://github.com/citusdata/cstore_fdw/issues/168, different compression methods would be interest. 
Also, query table expressions like ROLLUP, GROUPING SETS and CUBE are a bit slow in cstore_fdw so it would be nice if it could work better with that sort of expressions or resolve a greater part of the query, not just the loop up.

Thanks!

**Comment by Hengjie** [comment]
Yeah, agree with above. I'd love to have pushdown support of sum, count, group bys support so it doesn't do the aggregations at the pg level.
